db/migrate/20240615110144_create_and_populate_join_table_constituencies_parliaments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateAndPopulateJoinTableConstituenciesParliaments < ActiveRecord::Migration[7.1]
+  def change
+    create_join_table :constituencies, :parliaments do |t|
+      t.string :constituency_external_id
```

Comment:
This adds a third column - the join table should just have two columns. If you can't get the right settings by changing the options, just use the regular `create_table` with an `id: false` option.

Reply:
Now we are running regular updates on the import, this can be dropped. I'd originally included it in case constituency ids changed and we had to index on `external_id`